nucypher/policy/conditions/offchain.py

```diff
@@ -148,6 +148,25 @@ def query_response(self, data: Any) -> Any:
 
         return result
 
+    @staticmethod
+    def _process_result_for_eval(result: Any):
+        # strings that are not already quoted will cause a problem for literal_eval
+        if isinstance(result, str):
+            # check if already quoted; if not, quote it
+            if not (
+                (result.startswith("'") and result.endswith("'"))
+                or (result.startswith('"') and result.endswith('"'))
```

Comment:
`startswith` and `endswith` can accept tuples which can simplify this slightly.  Also, if you like to reduce further you can use ternary operation to determine the quote type.

```python
    if isinstance(result, str) and not (result.startswith(("'", '"')) and result.endswith(("'", '"'))):
        quote = '"' if "'" in result else "'"
```

Reply:
✔️ - https://github.com/nucypher/nucypher/pull/3553/commits/963c1312dd1de15da28d54e5434d59e83f7b8985